e2e/cypress/integration/api/iam/02_policies_api.spec.ts

```diff
@@ -104,68 +107,72 @@ describeIfIAMV2('policies API', () => {
     });
   });
 
-  describeIfIAMV2p1('project assignment enforcement', () => {
+  describe('project assignment enforcement', () => {
```

Comment:
Is this OK because it is nested in a `describeIfIAMV2p1` above?

Reply:
yep!